shared/query_engine/expressions/expressions.py

```diff
@@ -46,6 +51,9 @@ def __init__(self,
         self.right_raw = right_raw
         self.compare_op_raw = compare_op_raw
 
+        self.member = member
+        self.project = project
```

Comment:
Added these two properties to the Expression class, I think there may be some more cleanup that can be done now that we have these available